rsocket/RSocketClient.cpp

```diff
@@ -44,25 +45,30 @@ const std::shared_ptr<RSocketRequester>& RSocketClient::getRequester() const {
   return requester_;
 }
 
-folly::Future<folly::Unit> RSocketClient::resume() {
+folly::Future<folly::Unit> RSocketClient::resume(folly::EventBase* smEvb) {
```

Comment:
Why does this take the EventBase, wouldn't it just be passed in the constructor?